src/Components/bookHub/BookCard.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+
+import './BookCard.css';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+export default function Card({
+  id, title, author, image,
+}) {
+  const [authorVar, setAuthor] = useState();
+
+  const authorRecords = () => {
```

Comment:
same here, if you rename the function to something like 'getAuthor' then the code will be easier to read!
```suggestion
  const getAuthor = () => {
```